wbooks/src/app/components/itemDetail/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Text, View, Image } from 'react-native';
+import Button from '@components/button';
```

Comment:
Change name of the component to CustomButton

Reply:
the name of the Button component is changed to CustomButton